drivers/driver_linux.go

```diff
@@ -59,6 +59,7 @@ var (
 		"btrfs",
 		"zfs",
 		"vfs",
+		"ostree",
```

Comment:
There's no call to the `Register()` function with this name, so I don't think this does anything.

Reply:
this chunk must be deleted as now I am not adding a new driver but only an option for the overlay driver.

Thanks for spotting it!